<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_window</class>
 <widget class="QMainWindow" name="main_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>EC++ IDE</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(37, 37, 38);</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="central_widget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="mainLayout">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>14</number>
        </property>
        <property name="bottomMargin">
         <number>7</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="lbTamano">
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="styleSheet">
             <string notr="true">
             #lbTamano {
             font: 10pt &quot;MS Reference Sans Serif&quot;;
             color: rgb(183, 183, 183);
             }
             </string>
            </property>
            <property name="text">
             <string>Tama√±o de Letra</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QPushButton" name="btnMenos">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>24</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">
               #btnMenos {
               border: 1px solid rgb(183,183, 183);
               background-color: rgb(30, 30, 30);
               font: 10pt &quot;MS Reference Sans Serif&quot;;
               color: rgb(183, 183, 183);
               }
               #btnMenos:hover {
               border: 1px solid rgb(9, 145, 229);
               background-color: rgb(9, 145, 229);
               font: 10pt &quot;MS Reference Sans Serif&quot;;
               color: rgb(255, 255, 255);
               }
               </string>
              </property>
              <property name="text">
               <string>-</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnMas">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>24</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">
               #btnMas {
               border: 1px solid rgb(183,183, 183);
               background-color: rgb(30, 30, 30);
               font: 10pt &quot;MS Reference Sans Serif&quot;;
               color: rgb(183, 183, 183);
               }
               #btnMas:hover {
               border: 1px solid rgb(9, 145, 229);
               background-color: rgb(9, 145, 229);
               font: 10pt &quot;MS Reference Sans Serif&quot;;
               color: rgb(255, 255, 255);
               }
               </string>
              </property>
              <property name="text">
               <string>+</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="btnCompilar">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>38</height>
           </size>
          </property>
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;Ejecuta el programa&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="whatsThis">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Hola&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="styleSheet">
           <string notr="true">
           #btnCompilar {
           border: 1px solid  rgb(183,183, 183);
           background-color: rgb(30, 30, 30);
           font: 12pt &quot;MS Reference Sans Serif&quot;;
           color: rgb(183, 183, 183);
           }
           #btnCompilar:hover {
           border: 1px solid  rgb(9, 145, 229);
           background-color: rgb(9, 145, 229);
           font: 12pt &quot;MS Reference Sans Serif&quot;;
           color: rgb(255, 255, 255);
           }
           </string>
          </property>
          <property name="text">
           <string>Compilar</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnGuardar">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>38</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">
           #btnGuardar {
           border: 1px solid  rgb(183,183, 183);
           background-color: rgb(30, 30, 30);
           font: 12pt &quot;MS Reference Sans Serif&quot;;
           color: rgb(183, 183, 183);
           }
           #btnGuardar:hover {
           border: 1px solid  rgb(9, 145, 229);
           background-color: rgb(9, 145, 229);
           font: 12pt &quot;MS Reference Sans Serif&quot;;
           color: rgb(255, 255, 255);
           }
           </string>
          </property>
          <property name="text">
           <string>Guardar</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnAbrir">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>38</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">
           #btnAbrir {
           border: 1px solid  rgb(183,183, 183);
           background-color: rgb(30, 30, 30);
           font: 12pt &quot;MS Reference Sans Serif&quot;;
           color: rgb(183, 183, 183);
           }
           #btnAbrir:hover {
           border: 1px solid  rgb(9, 145, 229);
           background-color: rgb(9, 145, 229);
           font: 12pt &quot;MS Reference Sans Serif&quot;;
           color: rgb(255, 255, 255);
           }
           </string>
          </property>
          <property name="text">
           <string>Abrir</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnNuevo">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>38</height>
           </size>
          </property>
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;Ejecuta el programa&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="whatsThis">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Hola&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="styleSheet">
           <string notr="true">
           #btnNuevo {
           border: 1px solid  rgb(183,183, 183);
           background-color: rgb(30, 30, 30);
           font: 12pt &quot;MS Reference Sans Serif&quot;;
           color: rgb(183, 183, 183);
           }
           #btnNuevo:hover {
           border: 1px solid  rgb(9, 145, 229);
           background-color: rgb(9, 145, 229);
           font: 12pt &quot;MS Reference Sans Serif&quot;;
           color: rgb(255, 255, 255);
           }
           </string>
          </property>
          <property name="text">
           <string>Nuevo</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="txtCodigo">
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">
         #txtCodigo {
         border: 1px solid red;
         }
         </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lbEspacio">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="txtCmd">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>80</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">
         #txtCmd {
         border: 1px solid rgb(183,183, 183);
         background-color: rgb(30, 30, 30);
         font: 9pt &quot;Consolas&quot;;
         color: rgb(183,183, 183);
         }
         </string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="lbInput">
          <property name="styleSheet">
           <string notr="true">
           #lbInput {
           background-color: rgb(30, 30, 30);
           font: 10pt &quot;Consolas&quot;;
           color: rgb(200,200,200);
           }
           </string>
          </property>
          <property name="text">
           <string>&gt;&gt;&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lnInput">
          <property name="styleSheet">
           <string notr="true">
           #lnInput {
           border: 1px solid rgb(183,183, 183);
           background-color: rgb(30, 30, 30);
           font: 9pt &quot;Consolas&quot;;
           color: rgb(200,200,200);
           }
           </string>
          </property>
          <property name="frame">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menu_bar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="status_bar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
